examples/filters.rs

```diff
@@ -112,9 +118,9 @@ impl RequestLogger {
 
 impl ContextFilter for RequestLogger {
     ///Count requests and log the path.
-    fn modify(&self, ctx: FilterContext, context: &mut Context) -> ContextAction {
+    fn modify(&self, _ctx: FilterContext, context: &mut Context) -> ContextAction {
         *self.counter.write().unwrap() += 1;
-        ctx.log.note(&format!("Request #{} is to '{}'", *self.counter.read().unwrap(), context.uri));
+        info!("Request #{} is to '{}'", *self.counter.read().unwrap(), context.uri);
```

Comment:
...or maybe not. That won't print by default, either. This is such an important part of the example. Maybe we should add a hit to the welcome message that they can run it with debug prints. What do you think?